hazelcast/src/main/java/com/hazelcast/config/DurableExecutorConfig.java

```diff
@@ -182,13 +187,36 @@ public DurableExecutorConfig setSplitBrainProtectionName(String splitBrainProtec
         return this;
     }
 
+    /**
+     * Gets if statistics gathering is enabled
```

Comment:
Better use something like `Returns {@code true} if`. Maybe you can omit the `@return` then if they will be the same.

Reply:
fixed.